qa/tasks/cephfs/xfstests_dev.py

```diff
@@ -299,3 +303,88 @@ def write_ceph_exclude(self):
 
         self.mount_a.client_remote.write_file(join(self.xfstests_repo_path, 'ceph.exclude'),
                                               xfstests_exclude_contents, sudo=True)
+
+
+    #
+    # Following are helper methods that launch individual and groups of tests
+    # from xfstests-dev repo that is ready.
+    #
+
+    # generic helper methods
+
+    def run_test(self, cmdargs, exit_on_err=False):
+        """
+        1. exit_on_err is same as check_status in terms of functionality, a
+           different name is used to prevent confusion.
+        2. exit_on_err is set to False to make sure all tests run whether or
+           not all tests pass.
+        """
+        cmd = 'sudo ./check ' + cmdargs
```

Comment:
@rishabh-d-dave 

Please add the `env DIFF_LENGTH=0 ` before the `./check` to make it easier to debug.

Reply:
Sure. Thanks for the suggestion. I would've missed it otherwise.  :)